#Request
bool ready
---
#Response
int32[] ids
geometry_msgs/Pose[]   pose_objects